### PR TITLE
yum: add Spanish translation

### LINKED ISSUE #7695: translation request: yum in Spanish
Create a yum.md in Spanish using pages/yum.md as a reference for translation.

### PR DESCRIPTION
**Pull request by GhostViz**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Resolves #7695

### CODE REVIEW

File: pages.es/linux/yum.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Instala un nuevo paquete respondiendo si a todas las preguntas:
+
+`yum -y install {{package}}`
+
+- Encuentra que paquete provee un archivo determinado:
+
+`yum provides {{command}}`
+
+- Elimina un paquete
```
Comment (blueskyson) [inline pages.es/linux/yum.md:18]:
```suggestion
- Elimina un paquete:
```

File: pages.es/linux/yum.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Elimina un paquete:
+
+`yum remove {{package}}`
+
+- Muestra las actualizaciones disponibles para los paquetes instalados:
+
+`yum check-update`
+
+- Actualice los paquetes instalados a las versiones más recientes disponibles:
```
Comment (navarroaxel) [inline pages.es/linux/yum.md:26]:
```suggestion
- Actualiza los paquetes instalados a las versiones más recientes disponibles:
```

File: pages.es/linux/yum.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# yum
+
+> Administrador de paquetes para RHEL, CentOS y Fedora (para versiones anteriores).
+> Más información: <https://man7.org/linux/man-pages/man8/yum.8.html>.
+
+- Instala un nuevo paquete:
+
+`yum install {{package}}`
+
+- Instala un nuevo paquete respondiendo si a todas las preguntas:
```
Comment (navarroaxel) [inline pages.es/linux/yum.md:10]: Please include the indications in parenthesis
```suggestion
- Instala un nuevo paquete respondiendo si a todas las preguntas (también trabaja con actualizaciones, util para actualizaciones automáticas):
```

File: pages.es/linux/yum.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# yum
+
+> Administrador de paquetes para RHEL, CentOS y Fedora (para versiones anteriores).
+> Más información: <https://man7.org/linux/man-pages/man8/yum.8.html>.
```
Comment (navarroaxel) [inline pages.es/linux/yum.md:4]: We use this source now, could you please update the English and Persian pages too?
```suggestion
> Más información: <https://manned.org/yum>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @GhostViz, I left some comments.

**Review by navarroaxel — APPROVED** [review]

File: pages.es/linux/yum.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# yum
+
+> Administrador de paquetes para RHEL, CentOS y Fedora (para versiones anteriores).
+> Más información: <https://manned.org/yum>.
```
Comment (EmilySeville7cfg) [inline pages.es/linux/yum.md:4]:
```suggestion
> Más información: <https://man7.org/linux/man-pages/man8/yum.8.html>.
```

We prefer manned.org in a general case but the [original page](https://github.com/tldr-pages/tldr/blob/main/pages/linux/yum.md) uses another manual source so it's better to use the same one I think.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]
LGTM ;)

**Pull request merged**